Code/Framework/AzQtComponents/AzQtComponents/Gallery/ComponentDemoWidget.cpp

```diff
@@ -99,7 +99,7 @@ ComponentDemoWidget::ComponentDemoWidget(QWidget* parent)
     sortedPages.insert("Typography", new TypographyPage(this));
 
     // Pages hidden in 1.25 release - unused components, still need work before being made public, or not interesting for external devs
```

Comment:
Since this is the only page that was hidden left, can you remove this comment while you're here?

Reply:
Done